src/reducers/index.js

```diff
@@ -61,7 +61,7 @@ export const getTriggersByKonnector = (state, konnectorSlug) => {
   const triggers = Object.keys(triggersInState).reduce((acc, key) => {
     const document = state.cozy.documents['io.cozy.triggers'][key]
     if (
-      (document.worker === 'konnector' || document.worker === 'client') &&
+      fromConnections.isKonnectorTrigger(document) &&
       get(document, 'message.konnector') === konnectorSlug
     ) {
       acc.push(document)
```

Comment:
this commit should be split. Once for the lint (that should not exist), once for the "fix".